benchmark/runner.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Runner script for comparing the performance of code changes
+# optional flags:
+#   -b  supply the "before" commit SHA. Defaults to HEAD~1
+#   -a  supply the "after" commit SHA.  Defaults to current branch
+#   -f  supply the file name that contains the code to benchmark. Defaults to 'benchmark/test_code.rb'
+#
+# Full example: ./benchmark/runner.sh -b 8dfe3c5 -a 21d51bb -f benchmark/testing.rb
```

Comment:
nitpick: `test_code.rb`

Reply:
I was trying to show an instance where the user specifies a _different_ file since `test_code` is used by default.